packages/runtime-common/index-writer.ts

```diff
@@ -66,6 +68,7 @@ export interface InstanceEntry {
   fittedHtml?: Record<string, string>;
   atomHtml?: string;
   types: string[];
+  displayNames: string[];
```

Comment:
this approach is not going to work when we need to consider all the display names for the entire adoption chain when we calculate the aggregates

Reply:
ah, nevermind--i see what is happening here. you are capturing the entire adoption chain of display names for each entry